nodes/config/ui_base.js

```diff
@@ -188,6 +190,20 @@ module.exports = function (RED) {
         }
     }
 
+    function onTypeRegistered (node, type) {
+        // reload nodes from user directory package.json
+        if (RED.settings?.userDir) {
+            try {
+                const contribs = getThirdPartyWidgets(RED.settings.userDir)
```

Comment:
Does this not restrict such that https://github.com/FlowFuse/node-red-dashboard/pull/1286/files would not work anymore?